verticaltabs.js

```diff
@@ -847,7 +847,7 @@ VerticalTabs.prototype = {
 
     for (let i = 0; i < this.visibleTabs.length; i++) {
       let tab = this.visibleTabs[i];
-      if (tab.label.toLowerCase().match(input_value) || this.getUri(tab).spec.toLowerCase().match(input_value)) {
+      if (tab.label && (tab.label.toLowerCase().match(input_value) || this.getUri(tab).spec.toLowerCase().match(input_value))) {
```

Comment:
I think this is a little too broad…  If there is no tab.label, we'll still want to check against the uri, won't we?